ql/src/test/org/apache/hadoop/hive/ql/io/orc/TestOrcFileStripeMergeRecordReader.java

```diff
@@ -34,7 +35,7 @@
 
 public class TestOrcFileStripeMergeRecordReader {
 
-  private final int DEFAULT_STRIPE_SIZE = 5000;
+  private static final int DEFAULT_STRIPE_SIZE = 5000;
```

Comment:
Out of the context of this ticket but: DEFAULT_STRIPE_SIZE seems a bit low to me.
We are currently using 64k as default in ORC: https://github.com/apache/orc/blob/main/java/core/src/java/org/apache/orc/OrcConf.java#L30